src/Sulu/Bundle/AdminBundle/Resources/js/containers/DeleteDependantsDialog/DeleteDependantsDialog.js

```diff
@@ -0,0 +1,223 @@
+                promise.abort();
+            });
+        }
+
+        this.cancelled = true;
+
+        if (!onCancel) {
+            return;
+        }
+
+        onCancel();
+    };
+
+    handleClose = () => {
+        const {onClose} = this.props;
+
+        if (!onClose) {
+            return;
+        }
+
+        onClose();
+    };
+
+    @action handleSnackbarCloseClick = () => {
+        this.showSnackbarError = false;
+    };
+
+    render() {
+        const {dependantResourcesCount} = this.props;
+
+        return (
+            <Dialog
+                cancelText={
+                    this.errored || this.cancelled || this.finished
+                        ? translate('sulu_admin.close')
+                        : translate('sulu_admin.cancel')
+                }
+                confirmDisabled={this.errored || this.cancelled || this.finished}
+                confirmLoading={this.inProgress}
+                confirmText={translate('sulu_admin.delete')}
+                error={this.error && this.showSnackbarError ? this.error : undefined}
+                onCancel={
+                    this.errored || this.cancelled || this.finished
+                        ? this.handleClose
+                        : this.handleCancel
+                }
+                onConfirm={this.handleConfirm}
+                onSnackbarCloseClick={this.handleSnackbarCloseClick}
+                open={true}
+                title={translate('sulu_admin.delete_dependants_warning_title', {
+                    count: dependantResourcesCount,
+                })}
+                warning={this.cancelled && this.showSnackbarError
+                    ? translate('sulu_admin.delete_dependants_cancelled_text')
+                    : undefined
+                }
+            >
+                {!this.inProgress && !this.cancelled && !this.finished && !this.errored && (
+                    <p>
+                        {translate('sulu_admin.delete_dependants_warning', {
+                            count: dependantResourcesCount,
+                        })}
+                    </p>
+                )}
+
+                {(this.inProgress || this.cancelled || this.finished || this.errored) && (
+                    <React.Fragment>
+                        <div className={styles.progressBar}>
+                            <ProgressBar
+                                max={dependantResourcesCount}
+                                style={this.errored
+                                    ? 'error'
+                                    : this.finished
+                                        ? 'success'
+                                        : this.cancelled
+                                            ? 'warning'
+                                            : 'progress'}
+                                value={this.errored && this.totalDeletedResources === 0
+                                    ? 1
```

Comment:
why not pass `0` here? 🤔

Reply:
Because the progress bar would be empty then